packages/Interface/cmake/Dependencies.cmake

```diff
@@ -4,4 +4,7 @@ TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   Kokkos
   Teuchos
   Tpetra
+
+  LIB_REQUIRED_TPLS
+  MPI
```

Comment:
Sure but then let's reflect that in the documentation.

We have been getting back and forth on this, whether MPI should be optional or required.  If the `Interface` package requires it, and all other packages providing solution transfer operator (`Meshfree` and `Discretization`) have a required dependency on the `Interface` package, then DTK pretty much requires MPI.

Reply:
You haven't addressed this comment.  I'd be fine resolving it in a later PR but I want us to open an issue to track it.

Reply:
I think it should be required for DTK as a whole. See #418